main.js

```diff
@@ -0,0 +1,12 @@
+const mobileMenu = document.querySelector('.mobile-menu');
+const menuHamburger = document.querySelector('.menu');
+
+mobileMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('menu-mobile-active');
+  menuHamburger.classList.remove('menu-hidden');
+});
+
+menuHamburger.addEventListener('click', () => {
+  menuHamburger.classList.toggle('menu-hidden');
+  mobileMenu.classList.toggle('menu-mobile-active');
```

Comment:
- [ ] [OPTIONAL] Great job implementing the change requested by previous reviewer however please make sure that when the mobile menu is active/displayed and we expand back to the desktop screen size the menu disappears. In other words mobile menu should never appear on desktop screen size for better user experience and consistency

<img width="1052" alt="Screenshot 2023-06-20 at 7 41 10 AM" src="https://github.com/Mehr-o-maah/Portfolio_2.github.io/assets/44798044/ca1cdbba-9815-4163-a2df-cbe0486e40dd">